### PR TITLE
[FM v0.10.0]--Add doc release note

### PR DESCRIPTION
**Pull request by Huanli-Meng**
### Motivation

Add a doc release note for Function Mesh v0.10.0 ([code PR #560](https://github.com/streamnative/function-mesh/pull/560/)).

### Modification
- Add Release note v0.10.0
- Update the `sidebar.json` file
- Update the **Overview** document

### CODE REVIEW

File: docs/overview/overview.md [hunk 1]
```diff
@@ -182,4 +182,5 @@ Figure 7. The Function Mesh architecture
   - [Release notes v0.6.0](/releases/release-note-0-6-0.md)
   - [Release notes v0.7.0](/releases/release-note-0-7-0.md)
   - [Release notes v0.8.0](/releases/release-note-0-8-0.md)
-  - [Release notes v0.9.0](/releases/release-note-0-9-0.md)
\ No newline at end of file
+  - [Release notes v0.9.0](/releases/release-note-0-9-0.md)
+  - [Release notes v0.9.0](/releases/release-note-0-10-0.md)
```
Comment (tpiperatgod) [inline docs/overview/overview.md:186]: "Release notes v0.9.0" => "Release notes v0.10.0"

File: docs/releases/release-note-0-10-0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+title: Release notes v0.10.0
+category: releases
+id: release-note-0-10-0
+---
+
+Here are some highlights of this release. For a full list of updates available for Release v0.10.0, check out the [Function Mesh change log](https://github.com/streamnative/function-mesh/releases/tag/v0.10.0).
+
+## Support using wget to download HTTP/HTTPS function packages
+
+In previous releases, if a function package uses the HTTP or HTTPS protocol, you cannot download the function package directly if you do not have an `admin` access right. In this release, Function Mesh supports downloading an HTTP or HTTPs function package using [wget](https://www.gnu.org/software/wget/) if you enable the enable the [init container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) when installing the Function Mesh Operator. For details, see [function CRD configurations](/functions/function-crd.md#packages).
```
Comment (tpiperatgod) [inline docs/releases/release-note-0-10-0.md:11]: "if you enable the enable the"

**Review by tpiperatgod — CHANGES_REQUESTED** [review]

File: docs/releases/release-note-0-10-0.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Function Mesh Worker service supports HPA and VPA specifications
+
+Function Mesh supports Kubernetes [Horizontal Pod Autoscaler (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-Pod-autoscale/) and [Vertical Pod Autoscaler (VPA)](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler), to automatically adjust capacity of your Pods to maintain steady, predictable performance at the lowest possible cost. In this release, the Function Mesh Worker service also supports HPA and VPA specifications. For details, see [customizable options](/function-mesh-worker/reference/customizable-option.md).
```
Comment (teachwrite) [inline docs/releases/release-note-0-10-0.md:15]:
```suggestion
Function Mesh supports Kubernetes [Horizontal Pod Autoscaler (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-Pod-autoscale/) and [Vertical Pod Autoscaler (VPA)](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) to automatically adjust capacity of your Pods to maintain steady, predictable performance at the lowest possible cost. In this release, the Function Mesh Worker service also supports HPA and VPA specifications. For details, see [customizable options](/function-mesh-worker/reference/customizable-option.md).
```

File: docs/releases/release-note-0-10-0.md (hunk 2 above)
Comment (teachwrite) [reply docs/releases/release-note-0-10-0.md:11]: Also, no "an" before "admin".

**Review by teachwrite — APPROVED** [review]
LGTM

**Comment by Huanli-Meng** [comment]
@tpiperatgod Thanks for your comments. Docs are updated accordingly. PTAL again. Thanks.

**Review by tpiperatgod — APPROVED** [review]
lgtm

**Pull request merged**